src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,32 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+// Data
+import commentsFromServer from '../../api/comments';
+
+// Types
+import { Post } from '../../Types/post';
+import { Comment } from '../../Types/comment';
+
+import { PostInfo } from '../PostInfo';
+
+function getCommit(postId: number): Comment[] {
```

Comment:
Consider renaming the function 'getCommit' to 'getComments' to better represent its purpose.